test/react-native/component.test.js

```diff
@@ -21,7 +21,7 @@ describe('App', () => {
     const query = gql`query people { allPeople(first: 1) { people { name } } }`;
     const data = { allPeople: { people: { name: 'Luke Skywalker' } } };
     const networkInterface = mockNetworkInterface({ request: { query }, result: { data } });
-    const client = new ApolloClient({ networkInterface });
+    const client = new ApolloClient({ networkInterface, addTypename: false });
```

Comment:
Why do this all over the place and default the network interface to supporting it?